src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,67 @@
+import { useParams } from 'react-router-dom';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink/PersonLink';
+
+type Props = {
+  people: Person[];
+};
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  const { personSlug } = useParams();
+  const selectedPerson = people.find(person => person.slug === personSlug);
```

Comment:
As you use only slug of the selected person, you can save it only without finding the whole object.